tools/docker-compose/supervisor.conf

```diff
@@ -32,6 +32,10 @@ command = make uwsgi
 autostart = true
 autorestart = true
 redirect_stderr=true
+stopwaitsecs = 1
```

Comment:
Without this incantation, I couldn't get supervisor to *actually* reliably kill uwsgi 😢

Reply:
Weird.